modules/programs/kitty.nix

```diff
@@ -218,6 +210,20 @@ in {
       '';
     };
 
+    home.activation.kittythemechecker = mkIf (cfg.theme != null)
```

Comment:
I thought about writing a test for this check but I'm not sure how to test this. Should this even be tested (most of the tests I found check that config files are setup correctly rather than checking activation logic)?